src/app.ts

```diff
@@ -86,6 +86,13 @@ app.use(
   express.static(path.resolve("node_modules/govuk-frontend/govuk/assets")),
 );
 
+const healthcheckRouter = express.Router();
+healthcheckRouter.get("/", (req, res) => {
+  logger.info("Healthcheck returning 200 OK.");
+  res.status(200).send("OK");
+});
+app.use("/healthcheck", healthcheckRouter);
```

Comment:
There's not really much need for the router indirection, but it doesn't really hurt (could just be `app.get("/healthcheck")`)